scripts/pyenv_setup.sh

```diff
@@ -71,23 +76,25 @@ install_pyenv() {
     echo "Installing Python (if missing) via pyenv"
     local pyenv_version
     pyenv_version=$(pyenv -v | awk '{print $2}')
-    python_version=$(xargs -n1 < .python-version)
+    python_version=$(xargs -n1 <.python-version)
+    # NOTE: Older pyenv does not have access to the latest Python we require
+    if [[ "$pyenv_version" < 1.2.26 ]]; then
```

Comment:
`1.2.26` is the most recent `pyenv` release. If someone tries to use `make setup-pyenv` we will require them to update `brew` and `pyenv`.